Libraries/LibM/math.cpp

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2018-2020, Andreas Kling <kling@serenityos.org>
+ * Copyright (c) 2018-2020, SerenityOS developers
```

Comment:
Instead of removing someone else from a copyright header, please add yourself to it :)